source/docs/software/wpilib-tools/path-planning/pathfinder/introduction.rst

```diff
@@ -6,4 +6,4 @@ A more advanced approach to autonomous is called “path planning”. Instead of
 
 Paths have position and velocity values. A path has a list of position and velocity values for the robot every x seconds (usually 0.02 seconds or less). The robot then attempts to follow these points, adjusting using PID control when needed.
 
-Many teams and organizations have created path planning libraries for FRC, but the officially supported library for WPILib is Pathfinder, which can be added as a vendor library by adding this vendor JSON using the **Manage Libraries->Install New Libraries (online)** function in Visual Studio Code (see the `3rd Party Libraries <../../getting-started/3rd-party-libraries.html>`__ article for more details): https://dev.imjac.in/maven/jaci/pathfinder/PathfinderOLD-latest.json
+Many teams and organizations have created path planning libraries for FRC, but the officially supported library for WPILib is Pathfinder, which can be added as a vendor library by adding this vendor JSON using the **Manage Libraries->Install New Libraries (online)** function in Visual Studio Code (see the `3rd Party Libraries <../../getting-started/3rd-party-libraries.html>`__ article for more details): https://imjac.in/dev/maven/frc/7194a2d4-2860-4bcc-86c0-97879737d875
```

Comment:
@JacisNonsense Is this the way you want this?

Reply:
Yup, earlier in 2019 I changed some infra around that required dev to be a subroute instead of a subdomain, so you can skip that redirect entirely if you so please. The vendordeps files themselves still reference `dev.imjac.in`, mind